plugins/quay/src/components/QuayTagPage/component.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useRepository, useTagDetails } from '../../hooks';
+import { useParams } from 'react-router';
+import { QuayTagDetails } from '../QuayTagDetails';
+import { ErrorPanel } from '@backstage/core-components';
+// import { QuayTagDetails } from '../QuayTagDetails';
+
+type QuayTagPageProps = Record<never, string>;
+
+// TODO: replace useAsync with tanstack query
```

Comment:
There's no `useAsync` in this component.

Reply:
This comment was just adding to the code debt